pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -2731,14 +2740,14 @@ func (out *Outputs) HasParameters() bool {
 	return out != nil && len(out.Parameters) > 0
 }
 
-const MainLogsArtifactName = "main-logs"
+const LogsSuffix = "-logs"
 
 func (out *Outputs) HasLogs() bool {
 	if out == nil {
 		return false
 	}
 	for _, a := range out.Artifacts {
```

Comment:
one minor thought, this might break stuff, as users might have named their artifacts `mythink-log`. Is there any other field we can use?

Reply:
i thought maybe adding some `IsLogs bool` field to the Artifact, or maybe some `Type ArtifactType` enum, just to differentiate between log artifact and other types.